Mathlib/Topology/MetricSpace/Kuratowski.lean

```diff
@@ -14,35 +14,35 @@ import Mathlib.Topology.Sets.Compacts
 /-!
 # The Kuratowski embedding
 
-Any separable metric space can be embedded isometrically in `ℓ^∞(ℝ)`.
+Any separable metric space can be embedded isometrically in `ℓ^∞(ℕ, ℝ)`.
+Any partially defined Lipschitz map into `ℓ^∞` can be extended to the whole space.
+
 -/
 
 
 noncomputable section
 
 set_option linter.uppercaseLean3 false
 
-open Set Metric TopologicalSpace
-
-open scoped ENNReal
+open Set Metric TopologicalSpace NNReal ENNReal
 
-local notation "ℓ_infty_ℝ" => lp (fun n : ℕ => ℝ) ∞
+local notation "ℓ^∞(" ι ") " => lp (fun i : ι => ℝ ) ∞
```

Comment:
Not sure if we need the space after `)` on the LHS. We don't need the space before `)` on the RHS.
```suggestion
local notation "ℓ^∞(" ι ") " => lp (fun i : ι => ℝ) ∞
```

Reply:
Yeah, the spaces are a pretty printing hint, and the notation should be `"ℓ^∞(" ι ")"`

Reply:
Given the `ℓ^∞(ι, ℝ)` notation I'm seeing in the comments, perhaps we should make it be usable notation?
```lean
local notation "ℓ^∞(" ι ", " E ")" => lp (fun i : ι => E) ∞
local notation "ℓ^∞(" ι ")" => lp (fun i : ι => ℝ) ∞
```
Should this be scoped notation so others can conditionally use it too in other modules?